### PR TITLE
Create node.js.yml

### PR DESCRIPTION
**Pull request by shsn1990s**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+
+      - run: npm install
+
+      - name: MongoDB in GitHub Actions
+        uses: supercharge/mongodb-github-action@1.7.0
+
+      - run: npm run test:unit
```
Comment (PatrickReynoldsCoding) [inline .github/workflows/node.js.yml:33]: Is this the format Github CI needs to work?

Assuming the ENV are saved on Githubs end and If CI: false the pull request doesnt allow you to merge?

**Review by PatrickReynoldsCoding — COMMENTED** [review]
Looks good. Would love to have a 5 min run down of how it works.

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (ak2022dev) [reply .github/workflows/node.js.yml:33]: Thanks for providing the overview for this, Saif. Happy to approve it :)

**Review by ak2022dev — APPROVED** [review]
Thanks for providing the overview for this, Saif. Happy to approve it.

**Review by PatrickReynoldsCoding — APPROVED** [review]
Looks great

**Review by ak2022dev — APPROVED** [review]

**Pull request merged**